src/services/drpcEvents/index.ts

```diff
@@ -197,26 +183,26 @@ export default class DrpcEvents {
       }
       //find corresponding query based on queryIdForResponse provided by responder,
       //this should match a query id in our db
-      const [queryRow] = await this.db.get('query', { id: params.queryIdForResponse, role: 'requester' })
+      const [queryRow] = await this.db.get('query', { id: params.id, role: 'requester' })
```

Comment:
this rename to it just being an 'id' is a bit confusing ngl...cuz this is id not the local request it, it's still the request id from the requester, right?

Reply:
this is just to match the format set by @doowt in the docs. I think it's ok as long as seen in the proper context